node/src/actors/node.rs

```diff
@@ -53,7 +53,8 @@ pub fn run(config: Arc<Config>, callback: fn()) -> Result<(), failure::Error> {
     SystemRegistry::set(inventory_manager_addr);
 
     // Start RadManager actor
-    let rad_manager_addr = RadManager::default().start();
+    let rad_manager_addr =
+        RadManager::with_proxies(config.connections.retrieval_proxies.clone()).start();
```

Comment:
Not sure if setting the proxies here will save them if the actor panics and then restarts. That's why we always use the `starting` method in `actor.rs` to read the config on actor startup.

Reply:
Well, now it doesn't matter because if any actor panics we always stop the node.